app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -0,0 +1,245 @@
+rectangle.setFill(Color.GREEN);
+
+
+RotateTransition transition = new RotateTransition(Duration.seconds(5), rectangle);
+transition.setFromAngle(0);
+transition.setToAngle(360);
+transition.setInterpolator(Interpolator.LINEAR);
+
+transition.play();
+```
+
+### Scale Transition
+The [ScaleTransition](javafxdoc:ScaleTransition) creates a scale animation over a `duration`. 
+This is done by updating the `scaleX`, `scaleY` and `scaleZ` properties of the `Node` at regular interval.
+
+![ScaleTransition](/assets/images/javafx/animation/transition/scale-example.gif)
+```java
+Circle circle = new Circle(150, 150, 50, Color.GREEN);
+
+ScaleTransition transition = new ScaleTransition(Duration.seconds(5), circle);
+transition.setToX(0.1);
+transition.setToY(0.1);
+transition.setInterpolator(Interpolator.LINEAR);
+
+transition.play();
+```
+
+### Stroke Transition
+The [StrokeTransition](javafxdoc:StrokeTransition) creates an animation, that changes the stroke color of a shape over a `duration`. 
+This is done by updating the `stroke` property of the `Shape` at regular intervals.
+
+![StrokeTransition](/assets/images/javafx/animation/transition/stroke-example.gif)
+```java
+Circle circle = new Circle(150, 150, 50, Color.GREEN);
+circle.setStrokeWidth(5);
+
+StrokeTransition transition = new StrokeTransition(Duration.seconds(5), circle);
+transition.setFromValue(Color.GREEN);
+transition.setToValue(Color.BLACK);
+transition.setInterpolator(Interpolator.LINEAR);
+
+transition.play();
+```
+
+### Translate Transition
+The [TranslateTransition](javafxdoc:TranslateTransition) creates a move/translate animation that spans its `duration`.
+This is done by updating the `translateX`, `translateY` and `translateZ` properties of the `Node` at regular interval.
+
+![TranslateTransition](/assets/images/javafx/animation/transition/translate-example.gif)
+```java
+Circle circle = new Circle(50, 50, 10, Color.GREEN);
+
+TranslateTransition transition = new TranslateTransition(Duration.seconds(5), circle);
+transition.setToX(200);
+transition.setToY(200);
+transition.setInterpolator(Interpolator.LINEAR);
+
+transition.play();
+```
+
+### Pause Transition
+The [PauseTransition](javafxdoc:PauseTransition) executes an `Animation.onFinished` at the end of its `duration`.
+
+### Parallel Transition
+The [ParallelTransition](javafxdoc:ParallelTransition) plays a list of animations in parallel.
+
+### Sequential Transition
+The [SequentialTransition](javafxdoc:SequentialTransition) plays a list of animations in sequential order. 
+It is not recommended to contain an `Animation`, which is not the last one, with `Duration.INDEFINITE` as this will block all later animations in the sequence.
+
+## Timeline
+A [Timeline](javafxdoc:Timeline) is used to define a free form `Animation` of any `WritableValue<T>`. It is helpful if none of the built-in transitions operate on the required properties.
+
+It consists of a sequential series of `KeyFrame`.
```

Comment:
You sure this is the way to have words with ending `s` for code blocks?
It visually looks kinda weird:
![image](https://github.com/java/devjava-content/assets/88111627/664539f1-fb72-44a4-a323-f86d0cdd1b71)

Reply:
I agree that it might look a bit weird. Do you have a better idea that somewhat preserves the plural? I could also imagine not using a codeblock and just saying "Keyframes"?

In this repository, this kind of formatting is used in [Converting foreach with if](https://dev.java/learn/refactoring-to-functional-style/foreachwithif/#foreach) (`String`s) and [Effects, Gradients and Animations](https://dev.java/learn/javafx/effects/#config-actions) (`MyShape`s though that seems to be an edge case that might be considering to change). So I don't think this is common here.

Anyways, maybe actual reviewers/maintainers have other opinions or suggestions.